### PR TITLE
Remove Check On App Name For Console And My Acoount

### PR DESCRIPTION
**Pull request by Shaaali**
### Purpose
This PR resolves wso2-enterprise/asgardeo-product/issues/9104

### Related Issues
- Issue wso2-enterprise/asgardeo-product/issues/9104

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by thivi — APPROVED** [review]

File: apps/console/src/features/applications/components/application-list.tsx [hunk 1]
```diff
@@ -374,8 +375,12 @@ export const ApplicationList: FunctionComponent<ApplicationListPropsInterface> =
                     const hasScopes: boolean = !hasRequiredScopes(featureConfig?.applications,
                         featureConfig?.applications?.scopes?.delete, allowedScopes);
 
+                    const isSuperTenant: boolean = (tenantDomain === AppConstants.getSuperTenant());
+                    const hasSystemAppName: boolean = (UIConfig.systemAppsIdentifiers.includes(app?.name));
+                    const isSystemApp: boolean = isSuperTenant && hasSystemAppName;
```
Comment (jeradrutnam) [inline apps/console/src/features/applications/components/application-list.tsx:380]:
```suggestion
                    const isSystemApp: boolean = isSuperTenant && (UIConfig.systemAppsIdentifiers.includes(app?.name));
```

**Review by thivi — APPROVED** [review]

**Comment by Shaaali** [comment]
Verified in Dev for non-supertenant users.
The Apps could be successfully deleted too
Screenshot from 2022-04-08 08-17-08

File: apps/console/src/features/applications/components/application-list.tsx [hunk 2]
```diff
@@ -374,8 +375,11 @@ export const ApplicationList: FunctionComponent<ApplicationListPropsInterface> =
                     const hasScopes: boolean = !hasRequiredScopes(featureConfig?.applications,
                         featureConfig?.applications?.scopes?.delete, allowedScopes);
```
Comment (ThaminduDilshan) [inline apps/console/src/features/applications/components/application-list.tsx:377]: Let's remove this empty line. Can only keep line before the return statement (380)

Comment (Shaaali) [reply apps/console/src/features/applications/components/application-list.tsx:377]: Resolved comment with the PR : 
https://github.com/wso2/identity-apps/pull/3253

**Pull request merged**